CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ Optional:
 $ cp .env.example .env
 
 # Start MySQL in docker
-$ docker compose up -d
+$ docker-compose up -d
```

Comment:
Docker bundles `compose` into `docker` now, so you can skip using the Python version.

Reply:
Interesting, under ubuntu 22.04 that command was unknown. I'll remove this change then

Reply:
Not all Linux distros install this for some reason, see https://docs.docker.com/compose/install/linux/